stl/inc/span

```diff
@@ -450,16 +452,19 @@ public:
     }
 
     template <size_t _Size, enable_if_t<_Extent == dynamic_extent || _Extent == _Size, int> = 0>
-    constexpr span(element_type (&_Arr)[_Size]) noexcept : _Mybase(_Size), _Mydata(_Arr) {}
+    constexpr span(type_identity_t<element_type> (&_Arr)[_Size]) noexcept : _Mybase(_Size), _Mydata(_Arr) {}

```

Comment:
Let's go ahead and revert this (per https://github.com/microsoft/STL/pull/506#issuecomment-589113194).
```suggestion
    constexpr span(element_type (&_Arr)[_Size]) noexcept : _Mybase(_Size), _Mydata(_Arr) {}
```